UI/PantryScreen.tsx

```diff
@@ -121,27 +143,39 @@ export const PantryScreen = ({navigation}: NavigationProps) => {
             renderRightActions(progress, dragX, onClick)
           }
           onSwipeableOpen={() => closeRow(index)}
-          rightThreshold={10}
+          rightThreshold={20}
           overshootRight={false}
           overshootLeft={false}
           ref={ref => (row[index] = ref)}>
           <View style={localStyles.itemBox}>
-            <Text style={localStyles.itemText}>{item.title}</Text>
+            <Text style={localStyles.itemText}>{item}</Text>
           </View>
         </Swipeable>
       </GestureHandlerRootView>
     );
   };
 
-  const deleteItem = ({
-    item,
-    index,
-  }: ListRenderItemInfo<{id: number; title: string}>) => {
-    console.log(item, index);
-    let a = listData;
-    a.splice(index, 1);
-    console.log(a);
-    setListData([...a]);
+  const deleteItem = async ({item}: {item: string}) => {
+    Alert.alert('Delete  Confirmation: ' + item);
```

Comment:
The item is getting deleted by itself in the background without actual confirmation, in this alert box we can give the user two options, one ok to delete and the other one to cancel in case they don't want to actually delete the item. Each option has a onPress action for each choice. Can you implement it to where if they user clicks ok, we call the API and update the list and if click cancel nothing happens ? . The docs shows how to set it up: https://reactnative.dev/docs/alert